src/components/home/Features.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import FeatureCard from './FeatureCard.vue'
+</script>
+
+<template>
+  <div class="flex flex-col w-full items-center justify-center space-y-5">
+    <h2 class="text-4xl font-bold">
+      ✨应用亮点✨
+    </h2>
+    <div class="grid grid-cols-1 md:grid-cols-2 w-full md:w-3/5">
+      <FeatureCard title="弹幕及弹幕云过滤">
+        <template #desc>
+          Animeko 从弹弹play, 其他弹幕网站, 以及 Animeko 弹幕服务获取弹幕，你也可以发送弹幕到 Animeko 的公益弹幕服务器，AI
+          团队将逐渐推进弹幕云过滤功能。
+        </template>
+        <template #default>
+          <svg
+            class="h-6 w-6 text-white" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor"
+            aria-hidden="true"
+          >
+            <path
+              stroke-linecap="round" stroke-linejoin="round"
+              d="M10.05 4.575a1.575 1.575 0 10-3.15 0v3m3.15-3v-1.5a1.575 1.575 0 013.15 0v1.5m-3.15 0l.075 5.925m3.075.75V4.575m0 0a1.575 1.575 0 013.15 0V15M6.9 7.575a1.575 1.575 0 10-3.15 0v8.175a6.75 6.75 0 006.75 6.75h2.018a5.25 5.25 0 003.712-1.538l1.732-1.732a5.25 5.25 0 001.538-3.712l.003-2.024a.668.668 0 01.198-.471 1.575 1.575 0 10-2.228-2.228 3.818 3.818 0 00-1.12 2.687M6.9 7.575V12m6.27 4.318A4.49 4.49 0 0116.35 15m.002 0h-.002"
+            />
+          </svg>
+        </template>
+      </FeatureCard>
+      <FeatureCard title="使用 bangumi 管理追番进度">
+        <template #desc>
+          只要有一个 bangumi 账号，你就可以通过 Animeko 一站式地管理追番的进度，标注想看/再看/看过的番剧，以及和网友们讨论剧集。当然，你也可以以游客的身份体验大部分功能。
```

Comment:
```suggestion
          只要有一个 Bangumi 账号，你就可以通过 Animeko 一站式地管理追番的进度，标注想看/再看/看过的番剧，以及和网友们讨论剧集。当然，你也可以以游客的身份体验大部分功能。
```

顺便给 Bangumi 添加一个 href?